docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,196 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+coala has an awesome testing framework so that you can write tests for bears
+with ease.
+
+What is LocalBearTestHelper
+---------------------------
+
+You can use the following to test your bears:
+
+- LocalBearTestHelper class
+- verify_local_bears
+
+.. note::
+
+    ``execute_bears`` is used by LocalBearTestHelper class.
+
+Understanding the LocalBearTestHelper class
```

Comment:
If you directly distinct the different testing functions provided by `LocalBearTestHelper` you don't need to explain what it does, it gets kind of intuitive imo^^